packages/shared-ux/page/analytics_no_data/impl/src/analytics_no_data_page.test.tsx

```diff
@@ -29,28 +30,86 @@ describe('AnalyticsNoDataPage', () => {
     jest.resetAllMocks();
   });
 
-  it('renders correctly', async () => {
```

Comment:
Previously, this test was only rendering the "loading" state of the component. I moved the existing tests into a `describe` block for this.